examples/feature_selection/plot_rfe_with_cross_validation.py

```diff
@@ -27,33 +31,55 @@
     random_state=0,
 )
 
-# Create the RFE object and compute a cross-validated score.
-svc = SVC(kernel="linear")
-# The "accuracy" scoring shows the proportion of correct classifications
+# %%
+# Model training and selection
+# ----------------------------
+#
+# We create the RFE object and compute the cross-validated scores. The scoring
+# strategy "accuracy" optimizes the proportion of correctly classified samples.
+
+from sklearn.feature_selection import RFECV
+from sklearn.model_selection import StratifiedKFold
+from sklearn.svm import SVC
 
 min_features_to_select = 1  # Minimum number of features to consider
+svc = SVC(kernel="linear")
+cv = StratifiedKFold(10)
+
 rfecv = RFECV(
     estimator=svc,
     step=1,
-    cv=StratifiedKFold(2),
+    cv=cv,
     scoring="accuracy",
     min_features_to_select=min_features_to_select,
 )
 rfecv.fit(X, y)
 
-print("Optimal number of features : %d" % rfecv.n_features_)
-
-n_scores = len(rfecv.cv_results_["split0_test_score"])
-cv_scores = np.vstack(
-    (rfecv.cv_results_["split0_test_score"], rfecv.cv_results_["split1_test_score"])
-).T
+print(f"Optimal number of features: {rfecv.n_features_}")
```

Comment:
This will probably change a bit but this is good as well. I found 4 features.
I think we can add some discussion that the errorbar overlap and the solutions are equivalent.